### PR TITLE
[process-agent] Fix exit parameters for process tasks

### PR DESCRIPTION
**Pull request by mbakht**
### What does this PR do?

The parameter list of the `__exit__` function was recently modified as they were not being directly used: https://github.com/DataDog/datadog-agent/pull/7499/files#diff-2bdcc25ad3b160e80d6d20029f9343c6f51997a3ea92d73900705b9b8c58a53dL157

But those parameters are part of the signature of the `__exit__` function for ContextManager.

Without the params, we see the following error:

```
  File "/usr/local/lib/python3.8/dist-packages/invoke/tasks.py", line 127, in __call__
    result = self.body(*args, **kwargs)
  File "/home/vagrant/go/src/github.com/DataDog/datadog-agent/tasks/process_agent.py", line 147, in build_dev_image
    ctx.run(
TypeError: __exit__() takes 1 positional argument but 4 were given
```

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details that should be tested during the QA.

### CODE REVIEW

**Review by brycekahle — APPROVED** [review]

**Review by shang-wang — APPROVED** [review]

**Review by mx-psi — DISMISSED** [review]
Sorry about that, LGTM!

File: tasks/process_agent.py [hunk 1]
```diff
@@ -160,6 +160,6 @@ def __enter__(self):
         print("created tempdir: {name}".format(name=self.fname))
         return self.fname
 
-    def __exit__(self):
+    def __exit__(self, _, _, _):
```
Comment (mx-psi) [inline tasks/process_agent.py:163]:
```suggestion
    def __exit__(self, _exception_type, _exception_value, _exception_traceback):
```

Comment (mbakht) [reply tasks/process_agent.py:163]: Thanks @mx-psi, will check and update the PR once the Gitlab incident is resolved.

**Review by mx-psi — APPROVED** [review]
Feel free to ignore the ARM failures, they are a known issue (we are working on it)

**Pull request merged**